dockerfiles/piraeus-server/Dockerfile

```diff
@@ -22,11 +22,14 @@ RUN mkdir /var/log/linstor-controller && \
 	 chmod -R 0775 /var/log/linstor-controller
 
 # satellite
-RUN wget https://packages.linbit.com/piraeus/lvm2_2.03.02-3+b1_amd64.deb -O /tmp/l.deb && { dpkg -i /tmp/l.deb; apt-get install -y -f; } && rm -f /tmp/l.deb && apt-get remove -y udev && apt-get clean
+# needs a newer lvm, debian bug #932433
+COPY preferences /etc/apt/preferences
+RUN echo 'deb http://deb.debian.org/debian/ bullseye main contrib' >> /etc/apt/sources.list && apt-get update
+RUN apt install -y -t testing lvm2 && apt-get remove -y udev && apt-get clean
```

Comment:
I'd combine these lines into a single `RUN`. From my experience `apt-get update` should always be in the same `RUN` statement as the actual `apt-get install`. Also I'd change from `apt install` to `apt-get install`, as I think they still warn about their unstable CLI.

Reply:
fine with me putting it on one line. Thanks for spotting the `apt`, that was just a mistake. `apt-get` for scripting